service/src/test/java/greencity/ModelUtils.java

```diff
@@ -2105,6 +2113,30 @@ public static List<GeocodingResult> getGeocodingResult() {
         return geocodingResults;
     }
 
+    public static AddressLatLngResponse getAddressLatLngResponse() {
+        return AddressLatLngResponse
+            .builder()
+            .latitude(50.5555555)
+            .longitude(50.5555555)
```

Comment:
please use different values for `latitude` and `longitude`

Reply:
changed